contrib/freestanding_lib/freestanding.py

```diff
@@ -0,0 +1,255 @@
+#!/usr/bin/env python3
+
```

Comment:
should this file contain a license header ?

Reply:
Yup